payments-core/src/main/java/com/stripe/android/model/parsers/ElementsSessionJsonParser.kt

```diff
@@ -148,12 +149,15 @@ internal class ElementsSessionJsonParser(
             parseLinkFlags(linkSettingsJson)
         } ?: emptyMap()
 
+        val linkConsumerIncentive = json?.let { LinkConsumerIncentiveJsonParser.parse(it) }
```

Comment:
Should we use the feature flag here? This will prevent errors in the JSON from failing the request.

Reply:
Good call!